### PR TITLE
[SASS] More deprecation fixes for Sass 3.0.0

### PR DESCRIPTION
**Pull request by bspeare**
# How to test

- `yarn install` 
- `yarn storybook`

### CODE REVIEW

File: src/scss/abstracts/_utilities.scss [hunk 1]
```diff
@@ -137,51 +137,47 @@
   font-size: $h2-font-size;
   line-height: 1.25;
   font-weight: $font-weight-medium;
+  // https://garyridgway.github.io/clampCalculator/?&minWidth=600&maxWidth=1310&minFont=1.95&maxFont=2.3
+  font-size: clamp(1.95rem, calc(0.7887vw + 1.6542rem), 2.3rem);
```
Comment (pyrello) [inline src/scss/abstracts/_utilities.scss:139]: Should we get rid of the other `font-size` declaration?

File: src/scss/components/cta_uids3.scss [hunk 2]
```diff
@@ -0,0 +1,121 @@
+@import '../abstracts/variables';
```
Comment (pyrello) [inline src/scss/components/cta_uids3.scss:1]: Why does this file have `_uids3` in the name when other files being moved over do not?

Comment (bspeare) [reply src/scss/components/cta_uids3.scss:1]: We need it there until https://github.com/uiowa/uiowa/issues/8234.  I'm fine with any other name, but there's already a newer `cta.scss`.

File: src/scss/abstracts/_utilities.scss (hunk 1 above)
Comment (bspeare) [reply src/scss/abstracts/_utilities.scss:139]: The first `font-size` was there as a fallback for older browsers.  We can probably remove it now?

Comment (pyrello) [reply src/scss/abstracts/_utilities.scss:139]: No worries if it is a valid use case. Just noticed it appeared to be a duplicate.

**Review by pyrello — APPROVED** [review]
Looks good. Tested this via uiowa/uiowa#8144.

**Comment by bspeare** [comment]
Thanks @pyrello!

**Pull request merged**